CrowdStrike/ember-toucan-core
# File input field re-selection fix

## 🚀 Description

This PR fixes the issue @simonihmig mentioned at https://github.com/CrowdStrike/ember-toucan-core/pull/177#pullrequestreview-1478885953, where re-selecting the same file after a removal does nothing.

---

## 🔬 How to Test

- Go to https://0093993e.ember-toucan-core.pages.dev/docs/components/file-input-field
- Remove the existing file
- Click Select Files
- Add a file
- Remove the file
- Re-add the **same file you did previously**
- Verify the file was re-added

---

## 📸 Images/Videos of Functionality

N/A
